app/infrastructure/kafka_consumer.py

```diff
@@ -91,6 +91,8 @@ class OrganizationConsumer(Organization):
     @classmethod
     def load_from_dict(cls, data):
         data["followers"] = log2p(data.get("followers", 0))
+        data["views"] = log2p(data.get("views", 0))
+        data["reuses"] = log2p(data.get("reuses", 0))
```

Comment:
I think reuses should be kept as is and don't need to be normalized? (same as datasets)